app/controllers/application_controller.rb

```diff
@@ -18,8 +18,13 @@ def logged_in?
   end
 
   def authenticate_user!
-    # TODO: redirect to login instead
-    head :unauthorized unless logged_in?
+    unless logged_in?
```

Comment:
Feels like we go backwards and forward with this lines. 
What's going on with this?

Reply:
just making this TODO I left before

`# TODO: redirect to login instead`